gr-qtgui/python/qtgui/azelplot.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Free Software Foundation, Inc.
```

Comment:
Please change the Copyright year to be 2020 for all new files

Reply:
All set.  Just pushed the change.